docs/resources/kafka_topic_v2.md

```diff
@@ -10,6 +10,136 @@ Definition of a Kafka Topic to be applied to a Kafka cluster.
 ```
 resource "instaclustr_kafka_topic_v2" "example" {
   partitions = 3
+  configs {
```

Comment:
can you add `x-terraform-field-name: config` to this property in API spec?
properties of array type is better named as singular noun in TF. `configs` -> `config`
more details https://instaclustr.atlassian.net/wiki/spaces/IDS/pages/2241658905/Developing+for+API+v2+in+a+Terraform+Compliant+manner

Reply:
approved as api is deployed in preprod
and this is not wrong functionally, just minor TF schema inconsistency